Dockerfile.arm64

```diff
@@ -0,0 +1,46 @@
+# =======================================
+# ===== Build image for the backend =====
+# =======================================
+FROM arm64v8/golang:1.13-alpine AS builder-backend
+
+# gcc and musl-dev are required for building go-sqlite3
+ADD https://github.com/multiarch/qemu-user-static/releases/download/v4.1.0-1/qemu-aarch64-static /usr/bin/
+RUN apk --no-cache add gcc musl-dev
+
+WORKDIR /go/src/app
+COPY . .
+
+# CGO_ENABLED=1 is mandatory for building go-sqlite3
+RUN cd cmd/authelia && GOOS=linux GOARCH=arm64 CGO_ENABLED=1 go build -tags netgo -ldflags '-w' -o authelia
+
+
+# ========================================
```

Comment:
this frontend section could be extracted in its own Dockerfile to avoid duplication. This also means changing the build command accordingly but it's not a big deal IMO.

https://stackoverflow.com/questions/53659993/docker-multi-stage-how-to-split-up-into-multiple-dockerfiles

Reply:
There's no saving here, as this has to be done three times as each build step is on it's own respective host.

Unless we decided to do all the build steps in a single run, which could actually cause other timeout issues with Travis, then it should stay as is.